test/functional/feature_governance.py

```diff
@@ -77,6 +77,19 @@ def check_superblock(self):
 
         assert_equal(payments_found, 2)
 
+    def have_trigger_for_height(self, sb_block_height):
+        count = 0
+        for node in self.nodes:
+            valid_triggers = node.gobject("list", "valid", "triggers")
+            valid_triggers_list = list(valid_triggers.values())
+            for i in range(len(valid_triggers)):
+                if json.loads(valid_triggers_list[i]["DataString"])["event_block_height"] != sb_block_height:
+                    continue
+                if valid_triggers_list[i]['AbsoluteYesCount'] > 0:
```

Comment:
nit: why not something like?

```suggestion
            for trigger in list(valid_triggers.values()):
                if json.loads(trigger["DataString"])["event_block_height"] != sb_block_height:
                    continue
                if trigger['AbsoluteYesCount'] > 0:
```

(I haven